config/hubs/azure.carbonplan.cluster.yaml

```diff
@@ -126,6 +126,13 @@ hubs:
                   mem_guarantee: 1950G
                   node_selector:
                     hub.jupyter.org/node-size: Standard_M128s_v2
+              - display_name: "GPU: NC16as T4 v3"
+                description: "~16 CPU, ~90G RAM"
+                kubespawner_override:
+                  mem_limit: null
+                  mem_guarantee: 90G
+                  node_selector:
+                    hub.jupyter.org/node-size: Standard_NC16as_T4_v3
```

Comment:
this will also need a different docker image. For now, you can set this as `docker pull pangeo/ml-notebook:master`. In parallel, we'll work on a second carbonplan image for our application.

Reply:
Just a note on this. The configurator doesn't (yet) support updating images listed in these profiles, so to update image tags for this profile would involve opening a PR to this repo.